meetings/2024-01-24.md

```diff
@@ -0,0 +1,174 @@
+* ...: CGs are often where different implementers come together to build a technology
+* ...: the specification work is split between the CG and the (yet to be formed) WG
+* ...: it is important to make our apps interoperable. In the weekly CG meeting, there is a big focus on specs, and I would like to propose that we use this weekly call as more of a weekly plenary
+* ...: more demos, more discussion, less spec PR discussion
+* ...: perhaps a rotation: one week is demos, one week is spec PRs, one week for roadmap (e.g., topic proposed by eP)
+* TbL: this sounds good. can we talk about relationships between Solid and nearby projects? (e.g., DWEB, exports: sync with external services iCloud, Google)
+* SC: There is a PR about workspace selection. For CGs, there is a rough understanding that they take on issues in the incubation space
+* ...: historically, we have always had a space for implementation and demos
+* ...: we should be careful not to overstep what other groups take on. If/when WG is formed, the CG shouldn't reproduce what the WG is doing. Similar for developer experience
+* ...: No strong point on meeting structure. We used to have panels, and over time we didn't have the numbers to keep those topics alive
+* ...: Would like the weekly mtg to stay, allows us to welcome people to the group and help them navigate their way into Solid
+* ...: If we can grow the other meetings (e.g., special topic meeting), that would be great, but even there the numbers are not there
+* VB: we do have demos every week in the CG meeting. Want the agenda to be a collaborative thing. Want to maintain the Wed CG meeting, wouldn't want to change the frequency or topics
+* eP: What Michiel is talking about touches on the next STM. The practitioner meeting (MON) is great for demos and use cases
+* ...: also the test suite and QA, for interop, test suite, it is important to leverage that in CG meetings
+* VB: +1 on QA.
+* TbL: when proposing the agenda as a PR, I find that hard to process. It would be more reasonable to do something similar with (e.g.) Google docs. Especially for reading the agenda on a mobile device
+
+### Show some use cases on the "About" page of the solidproject.org website
+URL: https://github.com/solid/solidproject.org/issues/813
+
+* Proposed by Maxime Lecoq
+* ML: We already indicated ([in this PR](https://github.com/solid/solidproject.org/pull/814)) that Solid can be used for groups (the home page was only referring to individuals before). The idea is to show that Solid can be used for organisations, governments, and for the data space ecosystem.
+* ...: For instance, at Startin'Blox we received feedback from people of the data space community (like GaiX, IDSA) who were wondering how Solid work for their use cases. I'm not an expert on Data Space but it looks to me that some of their uses cases (like contracts) can be "implemented" in Solid (using client-to-client standard).
```

Comment:
I'm not certain about the Gaia-X or IDSA changes nor websites, but they seem the most likely. It would be good if Maxime (@lecoqlibre) can confirm/correct!
```suggestion
* ...: For instance, at [Startin'Blox](https://startinblox.com/) we received feedback from people of the data space community (like [Gaia-X](https://gaia-x.eu/), [IDSA](https://www.idsalliance.org/)) who were wondering how Solid work for their use cases. I'm not an expert on Data Space but it looks to me that some of their uses cases (like contracts) can be "implemented" in Solid (using client-to-client standard).
```

Reply:
For IDSA I was referring to this one: https://internationaldataspaces.org/ @TallTed 

Startin'Blox and Gaia-X links are good.

Reply:
Updated. Thanks, @lecoqlibre!